Atualização da classe de modelo

fulanuc opened the pull request:
Atualização da classe que armazena os dados da página, uma vez pesquisados na interação com a API do facebook, que pesquisa as páginas da rede social

fulanuc committed:
Atualização da classe de modelo

Atualização da classe que armazena os dados da página, uma vez pesquisados na interação com a API do facebook, que pesquisa as páginas da rede social

erivanbarbosa reviewed (changes requested):
retirar a linha do include

retirar a palavra item do nome das variáveis.
Deve ser apenas name, category, id e location.

fulanuc committed:
Atualização da classe de modelo (PageItem)

Atualização da classe que armazena os dados da página, uma vez pesquisados na interação com a API do facebook, que pesquisa as páginas da rede social

fulanuc committed:
Atualização da classe de modelo (PageItem)

Atualização da classe que armazena os dados da página, uma vez pesquisados na interação com a API do facebook, que pesquisa as páginas da rede social

erivanbarbosa reviewed (changes requested):
Adicionar método set para o atributo GraphLocation

Usar nomes mais significativos no construtor da classe.

fulanuc commented:
https://developers.facebook.com/docs/reference/android/3.0/interface/GraphLocation/

fulanuc committed:
Atualização da classe de modelo (PageItem)

Atualização da classe que armazena os dados da página, uma vez pesquisados na interação com a API do facebook, que pesquisa as páginas da rede social

erivanbarbosa reviewed (changes requested):
Adicionar setGraphObject

fulanuc committed:
Atualização da classe de modelo (PageItem)

Atualização da classe que armazena os dados da página, uma vez pesquisados na interação com a API do facebook, que pesquisa as páginas da rede social
